integration/travis/run_integration.sh

```diff
@@ -51,16 +53,26 @@ if [ "$curl_error" = true ]; then
   cat ${PROJECT_DIR}/../scheduler/log/cook.log
   exit 1
 fi
-timeout 180s bash -c "wait_for_cook 22321" || curl_error=true
+
+# Start slave after master is running
+LIBPROCESS_IP=172.17.0.1 COOK_DATOMIC="datomic:free://localhost:4334/cook-jobs" COOK_PORT=12322 COOK_FRAMEWORK_ID=cook-framework-1 lein run ${PROJECT_DIR}/travis/${CONFIG_FILE} &
+
+timeout 180s bash -c "wait_for_cook 12322" || curl_error=true
 if [ "$curl_error" = true ]; then
   echo "$(date +%H:%M:%S) Timed out waiting for cook to start listening, displaying cook log"
   cat ${PROJECT_DIR}/../scheduler/log/cook.log
   exit 1
 fi
+timeout 180s bash -c "wait_for_cook 22321" || curl_error=true
+if [ "$curl_error" = true ]; then
+    echo "$(date +%H:%M:%S) Timed out waiting for cook to start listening, displaying cook log"
+    cat ${PROJECT_DIR}/../scheduler/log/cook.log
```

Comment:
Can we have these go to distinct log files?

Reply:
Done